UserApi/UserApi.UnitTests/Logging/LoggerAdapterTest.cs

```diff
@@ -0,0 +1,124 @@
+using System;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using UserApi.Common.Logging;
+
+namespace UserApi.Tests.Logging
+{
+    [TestFixture]
+    public class LoggerAdapterTests
```

Comment:
I don't think you need to write tests for this because there's no business logic here but rather framework. But kudos for finding a way to write it up!